product_docs/docs/biganimal/release/administering_cluster/04_customizing_policy_ definitions.mdx

```diff
@@ -0,0 +1,208 @@
+
+
+- **Allowed service ports list in Kubernetes cluster**:
+
+ ```js
+ ["5432",  "9402",  "443", "8080",
+   "9090",  "3000",  "8443",  "9443",  "9100", "9201", "8088"]
+ ```
+
+
+- **Allowed AppArmor profiles**:
+
+ ```js
+ ["runtime/default"]
+ ```
+ 
+
+- **Allowed capabilities**: 
+
+ ```js
+ ["FOWNER"]
+ ```
+
+
+- **Max allowed memory bytes in Kubernetes cluster**: 
+
+ `6Gi`
+
+
+- **Max allowed CPU units in Kubernetes cluster**:
+
+ `2` 
+
+
+- **Allowed host paths for pod in Kubernetes cluster**:
+
+ ```js
+ {
+  "paths": [
+    {
+      "pathPrefix": "/var/log",
+      "readOnly": false
+    },
+    {
+      "pathPrefix": "/var/lib/docker/containers",
+      "readOnly": true
+    },
+    {
+      "pathPrefix": "/",
+      "readOnly": true
+    },
+    {
+      "pathPrefix": "/sys",
+      "readOnly": true
+    },
+    {
+      "pathPrefix": "/proc",
+      "readOnly": true
+    },
+    {
+      "pathPrefix": "/var/run/docker.sock",
+      "readOnly": false
+    },
+    {
+      "pathPrefix": "/run/containerd/containerd.sock",
+      "readOnly": false
+    },
+    {
+      "pathPrefix": "/dev",
+      "readOnly": false
+    },
+    {
+      "pathPrefix": "/boot",
+      "readOnly": true
+    },
+    {
+      "pathPrefix": "/lib/modules",
+      "readOnly": false
+    },
+    {
+      "pathPrefix": "/usr",
+      "readOnly": true
+    },
+    {
+      "pathPrefix": "/etc",
+      "readOnly": true
+    }
+  ]
+ }
+ 
+ ```
+
+### Other recommendations from Microsoft Defender for Cloud 
+
+Microsoft Defender for Cloud (which now includes Azure Secure Center and Azure Defender) analyzes the configurations of your Azure resources to identify potential vulnerabilities.
+
+You might still see recommendations from Microsoft Defender for Cloud even after customizing your policies and remediating non-compliant resources. See the following sections to understand why Microsoft raises the following recommendations.  
+
+#### Restrict unauthorized network access
+
+- **Usage of host networking and ports should be restricted**
+
+ BigAnimal runs containers that use the node network namespace to monitor network traffic statistics of Kubernetes cluster worker nodes. To prevent traffic sniffing and configuration changes to the worker node system, BigAnimal has removed all security capabilities for those containers. 
+
+- **Virtual networks should be protected by Azure Firewall**
+
+ BigAnimal does not enable the Azure Firewall. Instead, BigAnimal uses Azure Network Security Group whitelists to specify permissible inbound and outbound traffic.
+
+ If your organization requires an Azure Firewall for compliance purposes, contact [support@biganimal.com](mailto:support@biganimal.com).
+
+#### Manage access and permissions
+
+- **Privileged containers should be avoided**
+
+ BigAnimal must run some containers in the privileged mode for control plane functionality, such as securing and monitoring the application.
```

Comment:
We should probably make a stronger statement here, something about how those privileged containers never have inbound network access, etc.

This comment goes for all of these recommendations, really - we should gesture towards compensating controls for each. "Running containers as root user should be avoided" is a good example of the compensating controls language.

Reply:
@moiznalwalla I think @ba-edb is asking a pattern like this:
```
Running containers as root user should be avoided, but to achieve some management functionalities like securing and monitoring the application, BigAnimal requires to run some containers in the privileged mode
```

Reply:
Committed changes in an attempt to address this comment.

Reply:
Can you link the commit? I don't see these comments reflected in the PR.